internal/bminventory/inventory.go

```diff
@@ -5635,6 +5672,58 @@ func (b *bareMetalInventory) validateKernelArguments(ctx context.Context, kernel
 	return nil
 }
 
+func normalizeRendezvousIP(ipPtr *string) *string {
+	if ipPtr == nil {
+		return nil
+	}
+	trimmed := strings.TrimSpace(swag.StringValue(ipPtr))
+	if trimmed == "" {
+		return nil
+	}
```

Comment:
Wouldn't the swagger validation prevent the value coming in with whitespace? Isn't it checking that the thing is already formatted like an ip?